packages/ember-htmlbars/tests/integration/binding_integration_test.js

```diff
@@ -1,200 +0,0 @@
-import Ember from 'ember-metal/core';
-import run from 'ember-metal/run_loop';
-import jQuery from 'ember-views/system/jquery';
-import EmberView from 'ember-views/views/view';
-import { Binding } from 'ember-metal/binding';
-import EmberObject from 'ember-runtime/system/object';
-import { computed } from 'ember-metal/computed';
-import ContainerView from 'ember-views/views/container_view';
-import compile from 'ember-template-compiler/system/compile';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import { registerHelper } from 'ember-htmlbars/helpers';
-
-import { set } from 'ember-metal/property_set';
-
-import { registerKeyword, resetKeyword } from 'ember-htmlbars/tests/utils';
-import viewKeyword from 'ember-htmlbars/keywords/view';
-
-var view, MyApp, originalLookup, lookup, originalViewKeyword;
-
-var trim = jQuery.trim;
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('ember-htmlbars: binding integration', {
-  setup() {
-    originalViewKeyword = registerKeyword('view',  viewKeyword);
-    originalLookup = Ember.lookup;
-    Ember.lookup = lookup = {};
-
-    MyApp = lookup.MyApp = EmberObject.create({});
-  },
-
-  teardown() {
-    Ember.lookup = originalLookup;
-
-    runDestroy(view);
-    resetKeyword('view', originalViewKeyword);
-    view = null;
-
-    MyApp = null;
-  }
-});
-
-QUnit.test('should call a registered helper for mustache without parameters', function() {
-  registerHelper('foobar', function() {
-    return 'foobar';
-  });
-
-  view = EmberView.create({
-    template: compile('{{foobar}}')
-  });
-
-  runAppend(view);
-
-  ok(view.$().text() === 'foobar', 'Regular helper was invoked correctly');
-});
-
-QUnit.test('should bind to the property if no registered helper found for a mustache without parameters', function() {
```

Comment:
tested in [content-test.js:73](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/content-test.js#L73)